first-analysis-steps/analysis-productions.md

```diff
@@ -10,41 +10,37 @@
 
 Running DaVinci locally can be great for testing an options file, but is rarely appropriate for creating the full set of ROOT ntuples needed for an analysis. Historically, people would submit their jobs to the grid: one can send off a large number of DaVinci jobs to be handled by batch computing resources (in certain cases, this is still useful - see the [next lesson](davinci-grid) for how to do this). However, this still has some drawbacks, in particular:
 
-* Large datasets (especially in run 3) would require a long period of monitoring your grid jobs
-* Computing resources can be wasted if multiple analyses are independently producing similar ntuples
-* Ntuples can be lost, or removed when analysists leave, which can be an issue for [analysis preservation](https://twiki.cern.ch/twiki/bin/view/LHCb/AnalysisPreservationReproducibility)
+* Large datasets (especially in run 3) would require a long period of monitoring your grid jobs.
+* Computing resources can be wasted if multiple analyses are independently producing similar ntuples.
+* Ntuples can be lost, or removed when analysts leave, which can be an issue for [analysis preservation](https://twiki.cern.ch/twiki/bin/view/LHCb/AnalysisPreservationReproducibility).
```

Comment:
This twiki link is deprecated and any general matter concerning AP&OD should point to DPA's page https://lhcb-dpa.web.cern.ch/lhcb-dpa/wp6/index.html.

Reply:
Thanks, fixed it.